.github/workflows/pr-build.yml

```diff
@@ -10,7 +10,11 @@ on:
 jobs:
   build:
     if: ${{ github.actor != 'dependabot[bot]' }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        node: [14, 16]
```

Comment:
line 34 s/b `node-version: ${{ matrix.node }}` and lines 44-46 s/b

```
key: ${{ runner.os }}-${{ matrix.node }}-yarn-${{ hashFiles('yarn.lock') }}
          restore-keys: |
            ${{ runner.os }}-${{ matrix.node }}-yarn-
```

Reply:
Good catch. Build with the changes is running now